tedana/utils.py

```diff
@@ -208,7 +209,21 @@ def dice(arr1, arr2, axis=None):
         dsi = np.zeros(arr_sum.shape)
     else:
```

Comment:
I can't tag the whole if statement above, but it can be removed now that you have the warning and filter below.